ui/desktop/electron-app/config/forge.config.js

```diff
@@ -20,7 +20,14 @@ module.exports = {
       'entitlements-inherit': './assets/macos/entitlements.plist',
       'signature-flags': 'library',
     },
+    asar: true,
   },
+  plugins: [
+    {
+      name: '@electron-forge/plugin-auto-unpack-natives',
```

Comment:
This plugin was suggested if you use any native node modules, see [here](https://www.electronforge.io/config/plugins/auto-unpack-natives).

It just automatically configures the `asar` settings rather than us manually doing it.

Reply:
@ZedLi new to `asar`.  Is it a compression-less file format?

Reply:
It just concatenates the files together, it's basically like how we bundle our frontend code